backend/src/apiserver/resource/resource_manager.go

```diff
@@ -49,11 +49,31 @@ import (
 
 // Metric variables. Please prefix the metric names with resource_manager_.
 var (
+	extraLabels = []string{
+		// display in which Kubeflow namespace the runs were triggered
+		"profile",
```

Comment:
What's the rationale for naming this `profile` instead of `namespace`?

Reply:
I chose this since it is better to understand that the profile will always be the namespace that Kubeflow uses for specific workflow runs, not a generic namespace from the K8s Cluster.